<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinSpecRemoteReadout</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,1">
   <item row="1" column="0">
    <widget class="QGroupBox" name="plot_groupBox">
     <property name="title">
      <string>Spectrum</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2"/>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="control_groupBox">
     <property name="title">
      <string>WinSpec Remote Client Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Acq Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="interrupt_pushButton">
        <property name="text">
         <string>Interrupt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="acq_time_doubleSpinBox"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="start_pushButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QComboBox" name="wl_calib_comboBox"/>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="save_h5_checkBox">
        <property name="text">
         <string>Save H5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Horizontal Axis</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="continuous_checkBox">
        <property name="text">
         <string>Continuous</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
